my-app/src/API/index.ts

```diff
@@ -0,0 +1,8 @@
+export const API_KEY: string = 'bb2b538ec7dc87a9a9ecaa9065c3baff';
+
+export const getListUrl = (lng: string, list: string): string =>
+  `https://api.themoviedb.org/3/movie/${list}?api_key=${API_KEY}&language=${lng}&page=1`;
+
+export const NOW_PLAYING_REQUEST_WORD = 'now_playing';
```

Comment:
`WORD` here is strange, to my mind, you can use smth more specific. If it's a path in url - so pls call it `path`.

Reply:
done